qutebrowser/javascript/webelem.js

```diff
@@ -102,13 +104,31 @@ window._qutebrowser.webelem = (function() {
         out.attributes = attributes;
 
         const client_rects = elem.getClientRects();
+
+        // Get location of frame and add it to element
+        // TODO How to generate a 0 object without this
+        let frame_offset_rect = null;
```

Comment:
Since you set this below in any case, this seems useless.

Reply:
js creates new scopes for if statements as well, so variables declared inside if statements get deleted upon their exit (unlike python). I'm not sure if there's a better way to do this though (since the var has to be declared outside the statement. (ternary operators aren't allowed per eslint).